cmd/notary/tuf.go

```diff
@@ -83,6 +83,12 @@ var cmdTUFVerifyTemplate = usageTemplate{
 	Long:  "Verifies if the data passed in STDIN is included in the remote trusted collection identified by the Globally Unique Name.",
 }
 
+var cmdWitnessTemplate = usageTemplate{
+	Use:   "witness [ GUN ] <role> ...",
+	Short: "Marks roles to be re-signed the next time they're published",
+	Long:  "Marks roles to be re-signed the next time they're published. If the role has no currently valid signatures, or is otherwise invalid, a new version is published. If a role has some valid signatures and is not otherwise invalid, new signatures are added without modifying the signed role data.",
```

Comment:
Non-blocking - should we modify this comment since for now, if the role is valid, the version is bumped?